pipelinewise/fastsync/mongodb_to_postgres.py

```diff
@@ -110,7 +110,7 @@ def sync_table(table: str, args: Namespace) -> Union[bool, str]:
 def main_impl():
     """Main sync logic"""
     args = utils.parse_args(REQUIRED_CONFIG_KEYS)
-    cpu_cores = utils.get_cpu_cores()
+    cpu_cores = args.tap.get('fastsync_parallelism', utils.get_cpu_cores())
```

Comment:
would be great to eliminate that so many duplicates in `x_to_y.py` files one day.

Reply:
Indeed, so many files with same code!
Soon Peter, soon!